grpc-ballerina/Module.md

```diff
@@ -0,0 +1,326 @@
+## Module Overview
+
+This module provides an implementation for setting up, connection and interacting with gRPC endpoints. 
+
+gRPC is an inter-process communication technology that allows you to connect, invoke and operate distributed heterogeneous applications as easily as making a local function call. The gRPC protocol is layered over HTTP/2 and It uses Protocol Buffers for marshaling/unmarshaling messages. This makes gRPC, highly efficient on wire and a simple service definition framework.
+
+When you develop a gRPC application the first thing you do is define a service definition using Protocol Buffers.
+
+### Protocol buffers
```

Comment:
Shall we change the titles into either title case or sentence case?

Reply:
will change to title case

Reply:
This has be changed in all the following titles.